doc/cephfs/fs-volumes.rst

```diff
@@ -152,6 +152,20 @@ Fetch the absolute path of a subvolume using::
 
     $ ceph fs subvolume getpath <vol_name> <subvol_name> [--group_name <subvol_group_name>]
 
+Fetch the metadata of a subvolume using::
+
+    $ ceph fs subvolume info <vol_name> <subvol_name> [--group_name <subvol_group_name>]
+
+The output format is json and contains fields as follows.
+
+* bytes_pcent: quota used in percentage if quota is set, else displays "no-limit"
+* bytes_quota: quota size in bytes if quota is set, else displays "no-limit"
+* bytes_used: subvolume size in bytes
+* created_at: time of creation of subvolume in string format of datetime object
```

Comment:
Will the user understand what is 'datetime object'?

Reply:
It's not required to be mentioned for user. I will remove it.

Reply:
Done